src/env-inl.h

```diff
@@ -80,8 +80,28 @@ inline uint32_t* IsolateData::zero_fill_field() const {
   return zero_fill_field_;
 }
 
-inline Environment::AsyncHooks::AsyncHooks() {
-  for (int i = 0; i < kFieldsCount; i++) fields_[i] = 0;
+inline Environment::AsyncHooks::AsyncHooks(v8::Isolate* isolate)
+    : isolate_(isolate),
+      fields_(),
+      uid_fields_() {
+  v8::HandleScope handle_scope(isolate_);
+
+  // kAsyncUidCntr should start at 1 because that'll be the id for bootstrap.
```

Comment:
bootstrap meaning all the code that runs before the first `uv_run()` is executed.